File: 3. Conditional Loops/1. Cpp/2. Intermediate/additionoftwonumbers.cpp
#include <bits/stdc++.h>
using namespace std;
#define FIO ios_base::sync_with_stdio(false), cin.tie(0), cout.tie(0)
#define ll long long
#define endl "\n"
int main() {
	FIO;
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	int a, b;
	cin >> a >> b;
	int sum = 0;
	sum = a + b;
	if (a <= 0 && b <= 0) {
		cout << "Addition can't be done" << endl;
	} else {
		cout << "Result is: " << sum << endl;
	}
	return 0;
}

File: 3. Conditional Loops/2. Java/2. Intermediate/additionoftwonumbers.java
import java.util.Scanner;
public class additionoftwonumbers {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int a,b;
        System.out.print("A: ");
        a = sc.nextInt();
        System.out.print("B: ");
        b = sc.nextInt();
        int sum = 0;
        sum = a+b;
        if(a<=0 && b<=0){
            System.out.println("Addition can't be done");
        }
        else{
            System.out.println("Result is : " + sum);
        }
        sc.close();
    }
}
